schema_forms/mammo_form.py

```diff
@@ -1,191 +1,142 @@
-    mammo_lesion_left_breast = SelectField("Location of skin lesion on left breast", choices =
+    fld_mammo_lesion_left_breast = SelectField("Location of skin lesion on left breast", choices =
                                            MammographyDict.mammo_lesion_left_breast_choice)
-    mammo_asso_feature_skin_retraction = SelectField("Associated Feature: Skin Retraction", choices =
+    fld_mammo_asso_feature_skin_retraction = SelectField("Associated Feature: Skin Retraction", choices =
                                                      MammographyDict.mammo_asso_feature_skin_retraction_choice)
-    mammo_asso_feature_nipple_retraction = SelectField("Associated Feature: Nipple Retraction", choices =
+    fld_mammo_asso_feature_nipple_retraction = SelectField("Associated Feature: Nipple Retraction", choices =
                                                        MammographyDict.mammo_asso_feature_nipple_retraction_choice)
-    mammo_asso_feature_skin_thickening = SelectField("Associated Feature: Skin Thickening", choices =
+    fld_mammo_asso_feature_skin_thickening = SelectField("Associated Feature: Skin Thickening", choices =
                                                      MammographyDict.mammo_asso_feature_skin_thickening_choice)
-    mammo_asso_feature_trabecular_thickening = SelectField("Associated Feature: Trabecular Thickening", choices =
+    fld_mammo_asso_feature_trabecular_thickening = SelectField("Associated Feature: Trabecular Thickening", choices =
                                                        MammographyDict.mammo_asso_feature_trabecular_thickening_choice)
-    mammo_asso_feature_axillary_adenopathy = SelectField("Associated Feature: Axillary Adenopathy", choices =
+    fld_mammo_asso_feature_axillary_adenopathy = SelectField("Associated Feature: Axillary Adenopathy", choices =
                                                          MammographyDict.mammo_asso_feature_axillary_adenopathy_choice)
-    mammo_asso_feature_architectural_distortion = SelectField("Associated Feature: Architectural Distortion", choices =
+    fld_mammo_asso_feature_architectural_distortion = SelectField("Associated Feature: Architectural Distortion", choices =
                                                   MammographyDict.mammo_asso_feature_architectural_distortion_choice)
-    mammo_asso_feature_calicifications = SelectField("Associated Feature: Calcification", choices =
+    fld_mammo_asso_feature_calicifications = SelectField("Associated Feature: Calcification", choices =
                                                      MammographyDict.mammo_asso_feature_calicifications_choice)
-    mammo_birad = SelectField("BI-RAD classification (if available)",  choices =  MammographyDict.mammo_birad_choice)
+    fld_mammo_birad = SelectField("BI-RAD classification (if available)",  choices =  MammographyDict.mammo_birad_choice)
+    submit_button = SubmitField('Submit Form')
 
+    def to_bson(self):
+        fld_prefix = 'fld_'
```

Comment:
does this means it takes away the fld_ prefix from each object to create the field name?